src/models/IssueTableStorage.gs

```diff
@@ -0,0 +1,127 @@
+// Node required code block
+const Storage_ = require('../Storage.gs').Storage_;
+// End of Node required code block
+
+/**
+ * @file Contains class used to persist IssueTable data and access it.
+ */
+
+/**
+ * Creates a new IssueTableIndex_ instance, which is used to persist IssueTables and related information.
+ * 
+ * @constructor
+ */
+function IssueTableIndex_() {
+  this.storage_ = new Storage_('jst_tables', PropertiesService.getScriptProperties() || {});
```

Comment:
Is getScriptProperties the right place to save this data?
I havent read up on this much - I get it shouldnt be user properties and should be available to all users of the same sheet. But is ScriptProperties the contect for all users of your add-on

Reply:
Will ready into it a little bit more, it might actually be the `getDocumentProperties` for this purposes.

Reply:
according to https://developers.google.com/apps-script/reference/properties/properties-service i will then move over to `getDocumentProperties`. Thx for catching it.